apps/web/src/components/Common/Providers/index.tsx

```diff
@@ -29,26 +36,39 @@ const queryClient = new QueryClient({
   defaultOptions: { queries: { refetchOnWindowFocus: false } }
 });
 
+/*
+ * This enables the required abilities: Attachments, Emoji Reactions, Replies, Read receipts
+ * Note: Config changes requires db version update in the XMTPProvider
+ */
+const baseContentTypeConfigs = [
+  attachmentContentTypeConfig,
+  reactionContentTypeConfig,
+  replyContentTypeConfig,
+  readReceiptContentTypeConfig
+];
+
 const Providers = ({ children }: { children: ReactNode }) => {
   return (
     <ErrorBoundary>
-      <ServiceWorkerProvider />
-      <LeafwatchProvider />
-      <Web3Provider>
-        <ApolloProvider client={lensApolloClient}>
-          <LensSubscriptionsProvider />
-          <QueryClientProvider client={queryClient}>
-            <PreferencesProvider />
-            <TbaStatusProvider />
-            <FeaturedGroupsProvider />
-            <LivepeerConfig client={livepeerClient} theme={getLivepeerTheme}>
-              <ThemeProvider attribute="class" defaultTheme="light">
-                <Layout>{children}</Layout>
-              </ThemeProvider>
-            </LivepeerConfig>
-          </QueryClientProvider>
-        </ApolloProvider>
-      </Web3Provider>
+      <XMTPProvider contentTypeConfigs={baseContentTypeConfigs} dbVersion={1}>
```

Comment:
IMO xmtp provider can be wrapped only on messages page, global provider will increase size or it will consume more memory, might lead to memory leak!